src/v/redpanda/application.cc

```diff
@@ -565,7 +565,7 @@ void application::start() {
 
     syschecks::systemd_message("Starting controller").get();
     controller->start().get0();
-
+    _deferred.emplace_back([this] { controller->shutdown_input().get(); });
```

Comment:
don't you need to update the comments too? 

can you explain why this is the case here.

Reply:
good point, fixed